config.ru

```diff
@@ -0,0 +1,15 @@
+require 'puppet_webhook'
+require 'logger'
+
+LOGGER = Logger.new('/var/log/puppet_webhook').freeze
+COMMAND_PREFIX = 'umask 0022;'.freeze
+PIDFILE = '/var/run/puppet_webhook/puppet_webhook.pid'.freeze
```

Comment:
Is this used later on through some magic?

Reply:
Fixed